.github/actions/gam-canary-rollback/canary-probe.mjs

```
@@ -0,0 +1,156 @@
+
+const log = (m) => process.stderr.write(m + '\n');
+const emit = (rec) => process.stdout.write(JSON.stringify(rec) + '\n');
+const sleep = (ms) => new Promise((r) => setTimeout(r, ms));
+
+// A config error is a REFUSE, not a healthy default and not silently a regression — the caller
+// must still treat it as "block promotion", and it is emitted with state:'refuse' so it never
+// gets read as `.healthy // false` == a proven-bad build.
+function refuseConfig(reason) {
+  log(`❌ ${reason}`);
+  emit({ healthy: false, state: 'refuse', reason });
+  process.exit(2);
```

ESLint

[error] 'process' is not defined.

(no-undef)